### DISCUSSION TITLE
URDB energy tiers have nonstandard units of kWh daily

### DISCUSSION THREAD

**Discussion by boxpower** [NREL/REopt-API-Analysis#164] [2 - Site & Utility Questions]
I get the following response when I select any of the residential rates in PG&E territory from the URDB database in the web tool for an optimization. The full error is below:
{"core_urdb.jl_249"=>[" URDB energy tiers have nonstandard units of kWh daily"]}

The following rate can be selected to replicate the error:
URDB Label = "6629567f3eceaf868507e7e2"
Rate: "Pacific Gas & Electric Co: E-TOU-C Residential Time of Use Baseline Region P"

**Comment by adfarth** [comment]
Currently, we are unable to model utility rates with units of "$/kWh/day" in REopt. However, you can create a Custom Rate in the REopt web tool (must be logged in to do this) to convert the utility rate (including the tier max usage limits) to the expected units of $/kWh/month.
